packages/blockchain-wallet-v4-frontend/src/modals/Lockbox/LockboxSetup/index.js

```diff
@@ -19,7 +19,7 @@ import SetupTypeStep from './SetupTypeStep'
 import SoftwareDownloadStep from './SoftwareDownloadStep'
 
 class LockboxSetupContainer extends React.PureComponent {
-  componentWillMount () {
+  UNSAFE_componentWillMount () {
```

Comment:
maybe to replace it with `getDerivedStateFromProps` ?

Reply:
Well, the correct would be to refactor this component to a `FC`, and get rid of these lifecycles. I think that this is good for now, as we will probably refactor it in the future.